src/frontend/stories/chapter-preview.vue

```diff
@@ -28,11 +28,15 @@
       <ContentSidebar>
         <template #meta-box>
           <MetaBox
-            :created-at="props.chapter.createdAt"
-            :updated-at="props.chapter.updatedAt"
-            :story-type="props.meta.storyType"
-            :chapter-type="metaChapter"
-            :published-when="publishedWhen"
+            :primary="[
+              { label: props.meta.storyType, value: props.storyName },
```

Comment:
Props should be automatically spread into the template:

```suggestion
              { label: meta.storyType, value: storyName },
```